src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -59,12 +73,25 @@ def start_workunit(self, workunit):
     # Check if it is the first workunit
     first_span = not self._workunits_to_spans
     if first_span:
+      # If trace_id and parent_id are given create zipkin_attrs
+      if self.trace_id is not None:
+        zipkin_attrs = ZipkinAttrs(
+          trace_id=self.trace_id,
+          span_id=generate_random_64bit_string(),
+          parent_span_id=self.parent_id,
+          flags='0',
```

Comment:
Maybe add a comment here referencing where you got this value from, as it's non-obvious what it does